doc/content/lorawan/regional-parameters/eu868/_index.md

```diff
@@ -466,41 +121,11 @@ However, [ETSI](https://www.etsi.org/) (European Telecommunications Standards In
 
 Let’s have a look at how to calculate the time-on-air allowed per day (24 hours), per end device for some common duty cycles.
 
-
-<table>
-  <tr>
-   <td><strong>Duty cycle (take the maximum)</strong>
-   </td>
-   <td><strong>Equation: Time-On-Air = number of seconds per day </strong>X<strong> duty cycle</strong>
-   </td>
-   <td><strong>Maximum allowed Time-On-Air per day, per device</strong>
-   </td>
-  </tr>
-    <tr>
-   <td>0.1%
-   </td>
-   <td>86400 x 0.1%
-   </td>
-   <td>86 seconds per day
-   </td>
-  </tr>
-  <tr>
-  <tr>
-   <td>1%
-   </td>
-   <td>86400 x 1%
-   </td>
-   <td>864 seconds per day
-   </td>
-  </tr>
-   <td>10%
-   </td>
-   <td>86400 x 10%
-   </td>
-   <td>8640 seconds per day
-   </td>
-  </tr>
-</table>
+| Duty cycle (take the maximum) | Equation: Time-On-Air = number of seconds per day X duty cycle | Maximum allowed Time-On-Air per day, per device |
```

Comment:
Please remove equation from table heading, put it under or below the table, leave only "Time-On-Air" here

Reply:
Also, replace all notes like "take the maximum" from the table heading, or at least use "Max duty cycle" or something like that. It should be as short as possible. Pls apply this for all tables in this PR and request another review